tasks/Task.cpp

```diff
@@ -32,6 +88,27 @@ bool Task::startHook()
 void Task::updateHook()
 {
     TaskBase::updateHook();
+
+    base::samples::Joints cmd_in;
+    if (_cmd_in.read(cmd_in) != RTT::NewData) {
+        return;
+    }
+
+    for (auto const& command : cmd_in.elements) {
+        if (command.getMode() != m_cmd_in_mode) {
+            return exception(INVALID_COMMAND_MODE);
+        }
+    }
+
+    linux_pwms::PWMCommand output;
+    output.timestamp = base::Time::now(); // should this just fwd the cmd_in time?
```

Comment:
```suggestion
    output.timestamp = base::Time::now();
```
I think it is correct to base::Time::now(), as you are creating a new command.